include/libopencm3/stm32/common/i2c_common_v2.h

```diff
@@ -114,7 +114,17 @@ specific memorymap.h header before including this header file.*/
 #define I2C3_CR2			I2C_CR2(I2C3)
 #define I2C3_OAR1			I2C_OAR1(I2C3)
 #define I2C3_OAR2			I2C_OAR2(I2C3)
+
+#if defined(STM32G4) 
+/* NOTE: this implementation works for the G4 but might not work for other models. */
+/* TODO: check for possible implementation with other models */
+#define I2C3_RXDR			I2C_RXDR(I2C3)
+#define I2C3_TXDR			I2C_TXDR(I2C3)
```

Comment:
Just put this up with the rest of the "not all parts have i2c3" section please....